utopian-io/v2.utopian.io
# [UT-287] Separate steem account creation

### Verify that

This PR contains the base for Steem account creation separation from the signup.
Changes:
- Created a base layout for all the pages in the auth app (including the Steem creation ones):
![image](https://user-images.githubusercontent.com/6475893/49038021-e263fe00-f1a2-11e8-8840-3e8be3dccdb5.png)
- Implemented `hasClaimedBlockchainAccount` endpoint + tests + store actions
- Implemented route auth validation as @gregory-latinier did on the client app
- Created `steem/connect` page
- Created the base of `steem/create` page with the proper validations (auth & hasClaimed)


### **Test plan**

Route validation

* As a guest, go to `localhost:8081/en/steem/connect`

**Expected result:** You should be redirected to `localhost:8081/en/login`
___
Steem create page

* As a logged user, go to `localhost:8081/en/steem/connect`
* Click on "I don't have a steem account"
* You should be sent to `localhost:8081/en/steem/create`

**Expected result:** The page should mount and show the wip text 
___
Steem create page - user has claimed blockchain account
* Insert a new register in the db using your account data
```
db.utopianBlockchainAccounts.insertOne({ "blockchain": "steem", "address": "yourSteemAddress",  "provider": "github", "username": "yourGithubUsername" })
```
* Go to `localhost:8081/en/steem/create`
*  A modal should appear telling you can't create the account.
* Click on 'go back'

**Expected result:** you should be redirected to the main app
